agent/runtime-attach/build.gradle.kts

```
@@ -0,0 +1,11 @@
+plugins {
+  id("ai.java-conventions")
+}
+
+dependencies {
+  implementation("io.opentelemetry.contrib:opentelemetry-runtime-attach:1.14.0-alpha")
+  implementation("com.microsoft.azure:applicationinsights-agent:" + "3.2.11")
```

similar to https://github.com/microsoft/ApplicationInsights-Java/blob/7eb023e32f5ec4987c09f618f326ff92baf41815/smoke-tests/apps/SpringBootAttachInMain/build.gradle.kts#L6

```suggestion
  implementation(project(":agent:agent", configuration = "shadow"))
```